apps/expo-go/README.md

```diff
@@ -51,7 +51,8 @@ Metro needs to run prior running the build. Verify it runs on port 80. This is b
 For Android, run `./gradlew app:assembleDebug` in the `apps/expo-go/android` directory.
 
 For iOS:
-- set `DEV_KERNEL_SOURCE` to `LOCAL` in `EXBuildConstants.plist`
+- set `DEV_KERNEL_SOURCE` to `LOCAL` in `EXBuildConstants.plist`. You may need to copy `EXBuildConstants.plist.example` to `EXBuildConstants.plist` if it doesn't exist.
```

Comment:
this is a little strange. do you remember if there was any error from the `et ios-generate-dynamic-macros` step?

because internal the `et ios-generate-dynamic-macros` will generate the **EXBuildConstants.plist** file based on example

Reply:
The below `EACCES` error was happening in that script, and I fixed that error _after_ manually copying the plist. So this probably isn't needed?